src/trx-generator.ts

```diff
@@ -221,16 +222,18 @@ const renderTestSuiteResult = (
     const executionId = uuidv4();
     const fullTestName = path.basename(testSuiteResult.testFilePath);
     const time = new Date().toISOString();
+    const filepath = path.relative("./", testSuiteResult.testFilePath);
 
     // Failed TestSuite
     const unitTest = testDefinitionsNode
       .ele("UnitTest")
       .att("name", fullTestName)
-      .att("id", testId);
+      .att("id", testId)
+      .att("storage", filepath);
     unitTest.ele("Execution").att("id", executionId);
     unitTest
       .ele("TestMethod")
-      .att("codeBase", `Jest_${fullTestName}`)
+      .att("codeBase", filepath)
```

Comment:
I wonder if this is a breaking change. What do you think @deepakkamboj ?

Reply:
We have a similar change on line # 169 (.att("codeBase", filepath)), which is working fine. I have tested these changes locally and it is working fine.